packages/cli/generators/relation/has-one-relation.generator.js

```diff
@@ -106,7 +135,11 @@ module.exports = class HasOneRelationGenerator extends BaseRelationGenerator {
     );
 
     relationUtils.addProperty(sourceClass, modelProperty);
-    const imports = relationUtils.getRequiredImports(targetModel, relationType);
+    const imports = relationUtils.getRequiredImports(
```

Comment:
Any specific reason for this change please.

Reply:
It was by mistake (probably was working on sth else) Sorry about that.